handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -2,15 +2,22 @@
 
 We are looking for frontend engineers who know how to build intuitive user experiences to make the power of [Universal Code Search](https://about.sourcegraph.com/product) accessible to everyone. You will have a lot of ownership to solve tough technical and UX problems in key areas of our web application and browser extensions (e.g. code search, code navigation, campaigns).
 
+The following engineering teams are currently hiring for this role:
+
+1. [Web](../web/index.md#growth-plan) - all experience levels
+2. [Cloud](../cloud/index.md#growth-plan) - senior experience level
+
 ## About you
 
 - You have practice at creating high quality software balanced with a pragmatic understanding of how to make appropriate tradeoffs (e.g. cut scope) to ship quickly and iterate when necessary.
+- You are proficiency in the English language, both written and verbal, sufficient for success in a remote and largely asynchronous work environment.
```

Comment:
I think we can definitely delete this line, and can potentially delete others as well.

```suggestion
- You are proficiency in the English language, both written and verbal, sufficient for success in a remote and largely asynchronous work environment.
```

From our handbook:

https://about.sourcegraph.com/handbook/people-ops/hiring/job_description_guidelines#qualifications

> Avoid general skills that is expected of everyone we hire, or is expected for everyone in that role. For example, communication skills, being collaborative, able to write clean and reusable code for engineering roles, or writing skills for writer roles.

Reply:
@nicksnyder Thanks for pointing out the handbook guidance on this. I'm not 100% sure I agree with it completely yet, but that's a discussion for a different PR. To move this PR forward I've removed the reference to English language proficiency.

Reply:
@nicksnyder note that the link you shared is for the "Qualifications" section, but this is the "About you" section, which I had added to communicate to the candidate (not our recruiters) what we expect of them. I think it's very important to be explicit here, even repititive if necessary, because this is what candidates will read (not the linked handbook page). I do want to be able to be explicit here that we expect candidates to e.g. write "clean and reusable code" (like your linked section says to avoid in "Qualifications") or that they need to be able to communicate well in English as Jean proposed, because we actually get a lot of applications from candidates that do not meet those (seemingly obvious) criteria.